src/binary-reader-interp.cc

```diff
@@ -195,6 +195,7 @@ class BinaryReaderInterp : public BinaryReaderNop {
   wabt::Result OnTernaryExpr(wabt::Opcode opcode) override;
   wabt::Result OnUnreachableExpr() override;
   wabt::Result EndFunctionBody(Index index) override;
+  wabt::Result OnSimd1LaneImmExpr(wabt::Opcode opcode, Type type, uint32_t value) override;
```

Comment:
Why is this called `Simd1LaneImm`? I suppose it will be used for extract and replace lane, is that why?

Reply:
Yes, extract and replace lane only have 1 lane immediate parameter. But v8x16.shuffle have 16 lane Immediate parameters.